tests/python/test_k8s_properties_storages.py

```diff
@@ -102,7 +102,7 @@ def test_overwrite_config_map_storage(self):
         """
         pass
 
-    @attr('k8s', 'props', 'watch')
+    @attr('k8s', 'props', 'watch', 'infra')

```

Comment:
@ablatov it is app since legion.k8s.K8SConfigMapStorage is Legion specific component even if it extends k8s ConfigMap